ibis/expr/types/relations.py

```diff
@@ -2957,6 +2958,427 @@ def pivot_longer(
 
         return self.select(~pivot_sel, **new_cols)
 
+    @util.experimental
+    def pivot_wider(
+        self,
+        *,
+        id_cols: s.Selector | None = None,
+        names_from: str | Iterable[str | s.Selector] = "name",
+        names: Iterable[str] | None = None,
+        names_prefix: str = "",
+        names_sep: str = "_",
+        names_sort: bool = False,
```

Comment:
```suggestion
        names_from: str | Iterable[str | s.Selector] = "name",
        names_prefix: str = "",
        names_sep: str = "_",
        names_sort: bool = False,
        use_names: Iterable[str] | None = None,
```
??

Reply:
definitely not a blocker

Reply:
Hm, `use_names` sounds like a bool to me. Let's try with `names`, though I am +1 on moving the `names` argument to this position.